src/components/video/VideoCarousel.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+import { ref, watch } from 'vue';
+import { Button } from '../ui/button';
+import { Icon } from '@iconify/vue/dist/iconify.js';
+
+import VideoPlayer from './VideoPlayer.vue';
+
+type Video = {
+  link: string;
+  startTime?: number;
+  endTime?: number;
+};
+
+const props = defineProps<{
+  videos: Video[];
+}>();
+
+let currentVideoIndex = ref<number>(0);
+let isCurrentVideoFirst = ref<boolean>(true);
+let isCurrentVideoLast = ref<boolean>(false);
+
+watch(currentVideoIndex, () => {
+  isCurrentVideoFirst.value = currentVideoIndex.value == 0;
+  isCurrentVideoLast.value = currentVideoIndex.value == props.videos.length - 1;
+});
+
+function skipToNext() {
+  currentVideoIndex.value = Math.min(currentVideoIndex.value + 1, props.videos.length - 1);
+}
+
+function skipToPrev() {
+  currentVideoIndex.value = Math.max(currentVideoIndex.value - 1, 0);
+}
```

Comment:
nit: replace skip with go.
In this instance, we are not jumping over any videos, just going to the next / prev one.

Reply:
Accepted and implemented